sympy/calculus/singularities.py

```diff
@@ -31,3 +34,130 @@ def singularities(expr, sym):
                                   " implemented")
     else:
         return tuple(sorted(solve(simplify(1/expr), sym)))
+
+
+def range_func(func, set_value):
+    func = sympify(func)
+
+    if set_value.is_EmptySet:
+        return EmptySet()
+
+    free_symbol = func.free_symbols
+    if len(free_symbol) == 1:
+        symbol = free_symbol.pop()
+    elif len(free_symbol) == 0:
+        return FiniteSet(func)
```

Comment:
For `func = 1` which contains `no-symbols` FiniteSet(1) is to be returned if the domain is `non-empty`
`Use case`
```
>>> range_func(1, Interval(1, 2))
FiniteSet(1)
```
@hargup  please let me know if some thing is wrong here, or the point is not clear.